packages/next/src/client/app-index.tsx

```diff
@@ -20,6 +20,9 @@ import type { InitialRSCPayload } from '../server/app-render/types'
 import { createInitialRouterState } from './components/router-reducer/create-initial-router-state'
 import { MissingSlotContext } from '../shared/lib/app-router-context.shared-runtime'
 
+// Importing from dist so that we can define an alias if needed.
+import { findSourceMapURL } from 'next/dist/client/app-find-source-map-url'
```

Comment:
This import will be changed to a normal import from `src` when `findSourceMapURL` is implemented properly and we can remove the test mock.